src/components/slide_out_container.tsx

```diff
@@ -50,6 +48,10 @@ export class SlideOutContainer extends React.Component<SlideOutContainerProps, S
     </div>
 
   }
-}
+
+SlideOutContainer.defaultProps = {
+  showHeaderPanelArrow: true,
+  headerTextAlignment: 'left'
+};
 
```

Comment:
missing SlideOutContainer.displayName = 'SlideOutContainer'

Reply:
Also for many other components I see (this makes the snapshots know what the names of components are).